_data/banking.yml

```diff
@@ -98,6 +98,13 @@ websites:
       - hardware
     doc: https://www.bca.co.id/en/Individu/Produk/E-Banking/Klik-BCA
 
+  - name: Bank Hapoalim
+    url: https://www.bankhapoalim.co.il/
+    img: poalim.png
+    twitter: PoaliMarkeTweet
+    facebook: BankHapoalimSocial
```

Comment:
Can't find any page with that handle.

Reply:
Twitter: https://twitter.com/PoaliMarkeTweet
Facebook: https://www.facebook.com/BankHapoalimSocial

Both work for me.

Reply:
The thing is, the Facebook link doesn’t work for me or the script. ![image](https://user-images.githubusercontent.com/3535780/69835386-7f87c580-1241-11ea-9859-7c194f37a334.jpg)

Reply:
Perhaps they limited the page to Israeli users...